plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -71,7 +71,7 @@ func (m *FakePerformanceQuery) AddCounterToQuery(counterPath string) (PDH_HCOUNT
 	if c, ok := m.counters[counterPath]; ok {
 		return c.handle, nil
 	} else {
-		return 0, errors.New(fmt.Sprintf("AddCounterToQuery: invalid counter path: %s", counterPath))
+		return 0, fmt.Errorf("AddCounterToQuery: invalid counter path: %q", counterPath)
```

Comment:
To avoid the uppercase start of the error maybe we should output
```suggestion
		return 0, fmt.Errorf("in AddCounterToQuery: invalid counter path: %q", counterPath)
```
or change the wording altogether. Same below.

Reply:
Done for whole file